### ISSUE TITLE
[bug] ENA/SRA accessions not working

### ISSUE THREAD

**Issue by rpetit3** [bactopia/bactopia#116]
**Description**
ENA has made a change in their API endpoint.

Submitted fix to fastq-dl: https://github.com/rpetit3/fastq-dl/commit/9ab1f080ddd37ff9e68961aaf8fd69c449462ac0
will need as version bump

```
  2020-08-13 14:35:27:root:INFO - Query: SRX1720208
  2020-08-13 14:35:27:root:INFO - Archive: SRA
  Traceback (most recent call last):
    File "/opt/conda/envs/bactopia-gather_fastqs/bin/fastq-dl", line 419, in <module>
      logging.info(f'Total Runs To Download: {len(ena_data)}')
  TypeError: object of type 'bool' has no len()
```

**Steps to Reproduce**
```
bactopia --accession SRX1720208
```

**Expected Behavior**
Should download the FASTQ

**Execution Environment**

- Bactopia Version: 1.4.3
- OS: Debian
- Environment: conda/container

**Comment by rpetit3** [comment]
submitted release for fastq-dl: https://github.com/rpetit3/fastq-dl/releases/tag/v1.0.4

**Comment by rpetit3** [comment]
Fixed in [v1.4.4](https://github.com/bactopia/bactopia/releases/tag/v1.4.4)
